index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+     <div class="container">
+        <!-- header -->
+        <header>
+            <nav>           
+                 <h1>Alishah</h1>
+                 <a href="#"><img src="Image/Union.png" alt="button"></a>
+            </nav>
+        </header>
+
+        <!-- main -->
+        <div class="main-container">
+            <div class="title">
+                <h1>I'm Alishah. Glad to see you!</h1>
+            </div>   
+            <div class="decriptive">
+                <p>I’m AliShah.
+                    Glad to see you!
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+            </div>
+            <div class="social-media">
+                <span>LET’S CONNECT</span>
+            </div>
+            <div class="socail-media-icon">
+                <a href="#"><img src="./Image/Vector.png" alt=""></a>
+                <a href="#"><img src="./Image/Linkedin icon.png" alt=""></a>
+                <a href="#"><img src="./Image/Vector (3).png" alt=""></a>
+                <a href="#"><img src="./Image/Vector (1).png" alt=""></a>
+                <a href="#"><img src="./Image/Vector (4).png" alt=""></a>
+              
+            </div>
+         </div> 
+     </div>
```

Comment:
- [ ] **Good job so far 👏 👏** However, it would be better for this container to use a ```section``` tag instead of using ```<div>``` to represent a generic standalone section of the document, Kindly make this simple adjustment 🙏 [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 👍 👍